### ISSUE TITLE
gh re -l doesn't show private

### ISSUE THREAD

**Issue by gon138** [node-gh/gh#302]
It says listing all repos, but not including private repos. is there a way to list public and private all at once?

**Comment by wkoszek** [comment]
Same here. I'd expect --list to show everything.

**Comment by ivanperez-keera** [comment]
It's a bit confusing that neither of the following show private repos:
```
$ gh re --list --type all
$ gh re --list --type all --private
```

**Comment by henvic** [comment]
@ivanperez-keera have you tried `gh repo --list --private`?

Not sure if it works, but it might.

If not, I am happy to review and release a new version with such improvement, if someone sends me a pull request :)

**Comment by ivanperez-keera** [comment]
@henvic Yeap. `gh repo --list --private` only lists public repos.

I tried a few combinations and, without `--type private`, only public repos are listed.

**Comment by protoEvangelion** [comment]
Confirming that this bug still exists <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by codeallthethingz** [comment]
I tried a few variations and get an error message 

```bash
> gh repo --list -O myorg --type private
Listing private repos for myorg
fatal: You can only list your own public and private repos.
```

**Comment by protoEvangelion** [comment]
@codeallthethingz This seems to be working now

```
gh repo --list -u myorg --type private
```

**Comment by protoEvangelion** [comment]
Edit: Actually I take that back, it is including repos not from the organization passed in.

**Comment by protoEvangelion** [comment]
Running this for my organization returns all public & private repos:

```
gh repo --list -O myorg
```

I think the pagination [work that was recently committed](https://github.com/node-gh/gh/issues/539) & released in v1.12.14 allows this to function correctly. If you have a large organization you might have hit the page limit of 100 which only includes public repos. Now it properly returns all repos for the org passed in.
